client/src/pages/contact.tsx

```diff
@@ -1,16 +1,47 @@
 import * as React from 'react';
 import Layout from '../components/layout';
+import {FormattedMessage} from 'gatsby-plugin-intl';
 
 interface ContactPageProps {
   location: Location;
 }
 
 const ContactPage = ({location}: ContactPageProps) => {
+  const generalemail = 'screeningtool.feedback@usds.gov';
+  const techemail = 'screeningtool.support@usds.gov';
+
   return (<Layout location={location}>
     <section className={'usa-prose'}>
-      <h1>Contact</h1>
+      <h2><FormattedMessage
+        id={'contact.pageheader'}
+        description={'H2 header for contact page'}
```

Comment:
who sees this description? you can probably remove the html comments in there and just say `Contact Page Header` or something

Reply:
(side note, this is for translators, who may or may not have the full context of the page and may be seeing this string in isolation  - better to leave a semantic description of how it's used in a way that differentiates between the multiple senses of "contact" -- more on this here: https://www.transifex.com/blog/2020/tips-providing-context-translation/ , but something like "Heading for page to allow users to contact project maintainers" is a typical comment / description for this kind of string. A small point, and not something to change now necessarily, but we can talk with the translators and ask what they would find useful as well when it comes to this)

Reply:
Changed to `"Heading for page to allow users to contact project maintainers"`. 👍